microservices/webapp/vite-react/src/routes/Rooms.tsx

```diff
@@ -0,0 +1,247 @@
+import { NativeSelect, Table } from "@mantine/core"
+import { useState } from 'react';
+import { MultiSelect } from '@mantine/core';
+import { TextInput } from '@mantine/core';
+import { Room } from "../entities/Room";
+
+
+interface paramsObj {
+  name: string;
+  location: string;
+  description: string;
+  [key:string]: string;
+}
+
+
+const paramsObj: paramsObj = {
+  "name": "",
+  "location": "",
+  "description": "",
+  "type": ""
+}; 
```

Comment:
This is a little awkward, since we overwrite the `paramsObj` interface with a const instance. Instead, we can create items of type (interface) `paramsObj` using 
```
var foo = {
    "name": "bar",
    "location": "baz",
    "description": "fooDescription",
    "type": "room"
}
```

```suggestion
```